Modules/socketmodule.c

```diff
@@ -5419,6 +5426,7 @@ Return the service name from a port number and protocol name.\n\
 The optional protocol name, if given, should be 'tcp' or 'udp',\n\
 otherwise any protocol will match.");
 
+#ifndef __VXWORKS__
```

Comment:
Does an autoconf check work for getprotobyname?

Reply:
No